app/src/main/java/com/appunite/loudius/ui/pullrequests/PullRequestsScreen.kt

```diff
@@ -57,8 +63,13 @@ fun PullRequestsScreen(
     navigateToReviewers: NavigateToReviewers,
 ) {
     val state = viewModel.state
+    val swipeRefreshState = rememberPullRefreshState(
+        refreshing = state.data == Data.Loading,
+        onRefresh = { viewModel.fetchData() },
+    )
     PullRequestsScreenStateless(
         state = state,
+        pullRefreshState = swipeRefreshState,
```

Comment:
suggestion: maybe we will stick to `pullRefreshState` name instead of `swipeRefreshState` what do you think?